### ISSUE TITLE
To assess: is #1138 due to a bug or to very large data?

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-server#1140]
Ref:
* #1138

**Comment by jluethi** [comment]
Will have to look a bit more closely. It's one of the experiments from @adrtsc , right?

I think the size of this experiment is not a bug, but pushing the scale of experiments. I think it's something like 200 wells and multiple cycle (3-6?), combined with a workflow of ~10 tasks

2399 tasks done means the combination of nb_tasks * nb_images, right? (for tasks that parallelize on the image level)

=> would be great to check that we scale well to that level :)

**Comment by tcompa** [comment]
See https://github.com/fractal-analytics-platform/fractal-server/issues/474#issuecomment-1884422529.

Are 116 wells and 696 images and 13 tasks reasonable numbers? I see nothing obviously weird in there.

What could be a bug is not the number of wells/images, but rather the number of entries in the history. Each task (even when parallelized over N images) count as a single entry, and that's why 2000+ entries looks weird to me.

I'll also have to look further into this, maybe I'm missing something.

**Comment by jluethi** [comment]
> Each task (even when parallelized over N images) count as a single entry, and that's why 2000+ entries looks weird to me.

Ah ok, good question then!

> Are 116 wells and 696 images and 13 tasks reasonable numbers? I see nothing obviously weird in there.

The scale of the experiment is reasonable indeed

**Comment by tcompa** [comment]
We have deleted a few affected jobs from the DB, and are taking some mitigation strategies performance-wise (see https://github.com/fractal-analytics-platform/fractal-server/issues/1212#issuecomment-1932009350).

For the moment we are not fully exploring the issue, and the best guess is that _something_ went wrong due to repeated clicks form the web client (now mitigated in https://github.com/fractal-analytics-platform/fractal-web/pull/389, and with more mitigation in-progress at #1199).

Closing.
